tests/system/source/main.d

```diff
@@ -62,7 +62,7 @@ int main (string[] args)
         foreach (const ref addr; addresses)
             clients ~= new RestInterfaceClient!API(addr);
 
-        waitForDiscovery(clients, 5.seconds);
+        waitForDiscovery(clients, 10.seconds);
```

Comment:
🤦‍♂️

Reply:
Yeah this didn't fix problems either. sigh